website/docs/d/batch_pool.html.markdown

```diff
@@ -71,6 +91,51 @@ A `auto_scale` block exports the following:
 
 ---
 
+A `data_disks` block exports the following:
+
+* `lun` - The lun is used to uniquely identify each data disk.
+
+* `caching` - The caching mode of data disks.
+
+* `disk_size_gb` - The initial disk size in GB when creating new data disk.
+
+* `storage_account_type` - The storage account type to be used for the data disk.
+
+---
+
+A `disk_encryption_configuration` block exports the following:
+
+* `disk_encryption_target` - On Linux pool, only `TemporaryDisk` is supported; on Windows pool, `OsDisk` and `TemporaryDisk` must be specified.
+
+---
+
+An `extensions` block exports the following:
+
+* `name` - The name of the virtual machine extension.
+
+* `publisher` - The name of the extension handler publisher.The name of the extension handler publisher.
+
+* `type` - The type of the extensions.
+
+* `type_handler_version` - The version of script handler.
```

Comment:
shouldn't this be
```suggestion
* `script_handler_version` - The version of script handler.
```

Reply:
It is called 'Type Handler Version' in https://github.com/Azure/azure-rest-api-specs/blob/main/specification/batch/resource-manager/Microsoft.Batch/stable/2022-01-01/BatchManagement.json#L4587-L4589 
And there is a similar property in virtual_machine_extension resource. https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_machine_extension#type_handler_version
So I think we better keep it as type_handler_version to avoid confusion.